platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -227,6 +254,7 @@ class HangingProtocolService {
       // Viewport Settings
       //
       // protocol defined callback
+      viewport.viewportSettings = viewport.viewportSettings || { filter: () => [] };
```

Comment:
default viewportSettings can be just an array (no need to define `filter` as Arra.filter exist)